front/lib/api/invitation.ts

```diff
@@ -62,14 +63,16 @@ export async function getInvitation(
   return typeFromModel(invitation);
 }
 
-export async function updateInvitationStatus(
+export async function updateInvitationStatusAndRole(
   auth: Authenticator,
   {
     invitation,
     status,
+    role,
   }: {
     invitation: MembershipInvitationType;
     status: "pending" | "consumed" | "revoked";
+    role?: ActiveRoleType;
```

Comment:
Can it really be undefined

Reply:
When revoking it yes

Reply:
But I can modify the revoke logic to also send the `role` in the POST request